pkg/auth/defaultmanager.go

```diff
@@ -0,0 +1,30 @@
+package auth
+
+import (
+	"errors"
+	"log"
+
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+)
+
+var defaultKeyManager *KeyManager
+
+// DefaultKeyManagerWithConfig creates the default manager if it has not created yet.
+// This function must be called in main to make sure the default manager is created during service startup.
+// It will try to create the default manager only once even if called multiple times.
```

Comment:
for that, you could use something like `sync.Once.Do()`, as in https://github.com/fabric8-services/fabric8-common/blob/2cc2344d55776abe18178fc1c2efa8abc9996565/auth/token.go#L58-L63
(See also https://golang.org/pkg/sync/#Once.Do)

Reply:
See above.